presto-main/src/test/java/io/prestosql/operator/TestGroupedTopNBuilder.java

```diff
@@ -515,6 +515,7 @@ private long getEstimatedSizeInBytes()
     }
 
     // this class is for memory tracking comparison
+    @SuppressWarnings("unused")
     private static class TestPageReference
     {
```

Comment:
As the comment mention that this class is used for memory tracking so I didn't remove it here.